README.md

```diff
@@ -6,52 +6,35 @@
 Visit the [ReaPack website](https://reapack.com/) for ready-to-use binaries,
 the user guide or the package upload tool.
 
-## Building from source
+## Building
 
-Clone the repository and submodules:
+Prerequisites:
 
+- [vcpkg](https://vcpkg.io/en/index.html)
+- [CMake](https://cmake.org/) 3.15 or newer
+- C++17 compiler
+- PHP (Linux and macOS)
+
+    # 1. Clone repository with submodules
     git clone --recursive --shallow-submodules https://github.com/cfillion/reapack.git
 
-### Prerequisites
+    # 2. Create build directory
+    mkdir build; cd build
 
-Software requirements:
+    # 3. Install dependencies and prepare files for build system
+    VCPKG_ROOT=path\to\vcpcg cmake -DCMAKE_BUILD_TYPE=Release ..
 
-- [CMake](https://cmake.org/) 3.15 or newer
-- C++17 compiler (MSVC on Windows)
-- PHP (Linux and macOS only)
+    # 4. Build
+    make
 
-#### Linux
-
-Install the following libraries (and development headers if your system provides
-them separately):
-
-- [Boost](https://www.boost.org/) (1.56 or later)
-- [Catch2](https://github.com/catchorg/Catch2) (3.0 or later)
-- [libcurl](https://curl.haxx.se/libcurl/)
-- [libxml2](http://www.xmlsoft.org/)
-- [OpenSSL](https://www.openssl.org/) or compatible
-- [SQLite](https://www.sqlite.org/)
-- [zlib](https://www.zlib.net/)
+    # 5. Install ReaPack into your REAPER installation
+    cmake --target install
 
 #### macOS
 
-Install Boost and Catch2 using [Homebrew](https://brew.sh) (recommended).
 The build tools can be installed using `xcode-select --install` or the Xcode IDE.
 
-#### Windows
```

Comment:
Lots of instructions for beginners are being lost here...

Reply:
I believe that if a user wants to build a cross-platform app, they already know the tools to do so, otherwise READMEs would span a lot of lines. 
However, if you think this part is crucial, I may revert changes